### PR TITLE
Update plain_rules.md

### PR DESCRIPTION
**Pull request by mapedraza**
fix typo

### CODE REVIEW

File: documentation/plain_rules.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ on how to scape characters at
 
 ### Pre-SELECT clauses
 
-There are support for pre select clauses. Specifically we support `expression VAR for alias {myexpression}`. This allow
+There are support for pre select clauses. Specifically we support `expression VAR alias for {myexpression}`. This allow
```
[the remark this answers is not in the document]
Comment (AlvaroVega) [reply documentation/plain_rules.md:144]: alias for

**Review by AlvaroVega — APPROVED** [review]
LGTM

File: documentation/plain_rules.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (fgalan) [reply documentation/plain_rules.md:144]: NTC

**Review by fgalan — APPROVED** [review]
LGTM

**Pull request merged**